### PR TITLE
update graalvm version to 1.0.0-rc14 (#1566)

### PR DESCRIPTION
**Pull request by machi1990**
See #1566

### CODE REVIEW

**Comment by machi1990** [comment]
@Sanne this is ready for review.

File: core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java [hunk 1]
```diff
@@ -278,7 +278,7 @@ public void provideOutcome(AppCreator ctx) throws AppCreatorException {
             //TODO: use an 'official' image
             String image;
             if (dockerBuild.toLowerCase().equals("true")) {
-                image = "swd847/centos-graal-native-image-rc13";
+                image = "swd847/centos-graal-native-image-rc14";
```
Comment (Sanne) [inline core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java]: looks like these containers aren't available yet. We will have to wait for those before merging.

Comment (machi1990) [reply core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java]: 👍 Sure thing.

Comment (stuartwdouglas) [reply core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java]: I think @cescoffier was going to setup a quay.io image repo so we have an official place to host these.

Comment (machi1990) [reply core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java]: Good, thanks for the update @stuartwdouglas. I'll be following the updates.

File: core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java [hunk 2]
```diff
@@ -466,8 +466,10 @@ public void provideOutcome(AppCreator ctx) throws AppCreatorException {
     private boolean isThisGraalVMRCObsolete() {
```
Comment (machi1990) [inline core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java]: I have a small question @Sanne regarding this method. It is commented with `//FIXME remove after transition period` ? A quick search indicates that it's used only once in this file line `267` assigned to a boolean that is never used. What was the transition period? Are we still in it?

Comment (Sanne) [reply core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java]: Yes we still are in a "transition period".

By that I mean that we'll be following each latest GraalVM update. Eventually we'll have it depend on stable tags, then we'll have to revisit how we do this.

Comment (machi1990) [reply core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java]: Thanks for the response.

**Comment by Sanne** [comment]
hi @machi1990 , the migration to Quay was done: please see #1625 for details.

Could you update the name of the docker image, and rebase this please?

**Comment by machi1990** [comment]
> hi @machi1990 , the migration to Quay was done: please see #1625 for details.
> 
> Could you update the name of the docker image, and rebase this please?

Great. I'll check this right away as I also have some rebasing todo.

File: core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java (hunk 1 above)
Comment (machi1990) [reply core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java]: > looks like these containers aren't available yet. We will have to wait for those before merging.

Done. Ready for another review.

**Comment by Sanne** [comment]
building it once more locally so to check quickstarts too..

**Review by Sanne — APPROVED** [review]

**Comment by Sanne** [comment]
merged. Many thanks @machi1990 !

**Comment by cescoffier** [comment]
Thanks a lot @machi1990 ! That's awesome!

**Pull request merged**